edweiss-app/app/(app)/courses/[id]/index.tsx

```diff
@@ -143,162 +123,57 @@ const CoursePage: React.FC<LightDarkProps> = ({ light, dark }) => {
 			<RouteHeader title={course.data.name} align="center" isBold={true} />
 
 			{/* ScrollView pour permettre le défilement */}
-			<TScrollView contentContainerStyle={[styles.container, { backgroundColor }]} testID={testIDs.scrollView} >
+			<TScrollView p={16} backgroundColor="mantle" testID={testIDs.scrollView} >
+
 				{/* Section des Pending Assignments */}
-				<TText style={[styles.subHeader, { color: headingsColor }]} testID={testIDs.upcomingAssignments} >{t(`course:upcoming_assignment_title`)}</TText>
+				<TText mb={10} size={18} color='darkBlue' bold={true} testID={testIDs.upcomingAssignments} >{t(`course:upcoming_assignment_title`)}</TText>
 
 				{/* Liste des assignments avec map */}
-				{upcomingAssignments.length > 0 ?
+				{upcomingAssignments.length > 0
+					?
 					upcomingAssignments.map((assignment) => (
 						<TView key={assignment.name} testID={testIDs.assignemtView}>
-							<AssignmentDisplay item={assignment} index={upcomingAssignments.indexOf(assignment)} backgroundColor={backgroundColor} swipeBackgroundColor={swipeBackgroundColor2} defaultColor={defaultColor} swipeableRefs={swipeableRefs} />
+							<AssignmentDisplay item={assignment} index={upcomingAssignments.indexOf(assignment)} isSwipeable={true} />
```

Comment:
Maybe you can use a map instead of indexOf().

Reply:
Both options are ok imo